pkg/model/dev.go

```diff
@@ -99,6 +99,7 @@ type Dev struct {
 	Environment          []EnvVar              `json:"environment,omitempty" yaml:"environment,omitempty"`
 	Secrets              []Secret              `json:"secrets,omitempty" yaml:"secrets,omitempty"`
 	Command              []string              `json:"command,omitempty" yaml:"command,omitempty"`
+	Healthchecks         bool                  `json:"healthchecks,omitempty" yaml:"healthchecks,omitempty"`
```

Comment:
would this make them on or off by default?

Reply:
it disables them by default